src/components/RecentActivities/styles.css

```diff
@@ -0,0 +1,8 @@
+.recent-activity {
+    padding: 32px 0 !important;
+    background-color: #ffffff !important;
+}
+
+.activity-table {
+    padding: 0px !important;
+}
```

Comment:
I think this has been a conflict resolution after this branch was updated, please revert it back.